packages/ubuntu_desktop_installer/lib/pages/welcome/welcome_model.dart

```diff
@@ -55,6 +56,22 @@ class WelcomeModel extends SafeChangeNotifier {
   /// Returns the name of the language at the given [index].
   String language(int index) => _languageList[index].name;
 
+  /// Searches for a language matching the given [query]. The search is case-
+  /// insensitive, ignores diacritics, starts from the selected language, and
+  /// wraps around if not found.
+  int searchLanguage(String query) {
```

Comment:
hmm, we're going to need the same wrapping search logic for all lists. this could be shared somewhere

Reply:
Hmm, this would probably fit the collection of "non-visual" ubuntu_widgets better.